modules/virt-importing-vm-wizard.adoc

```diff
@@ -2,20 +2,31 @@
 //
 // * virt/virtual_machines/importing_vms/virt-importing-vmware-vm.adoc
 // * virt/virtual_machines/importing_vms/virt-importing-rhv-vm.adoc
-[id='virt-importing-vm-wizard_{context}']
-ifdef::virt-importing-vmware-vm[]
-= Importing a VMware virtual machine or template with the virtual machine wizard
 
-You can import a VMware virtual machine or template by using the virtual machine wizard.
-endif::[]
-ifdef::virt-importing-rhv-vm[]
-= Importing a Red Hat Virtualization virtual machine with the virtual machine wizard
+[id="virt-importing-vm-wizard_{context}"]
+= Importing a VM with the VM Import wizard
```

Comment:
s/VM/virtual machine/ for consistency
'VM Import' can stay as I'm guessing it's UI name